packages/lodestar/test/unit/network/util.ts

```diff
@@ -9,7 +8,7 @@ export async function createNode(multiaddr: string): Promise<NodejsNode> {
   const peerId = await createPeerId();
   const enr = ENR.createFromPeerId(peerId);
   const randomPort = Math.round(Math.random() * 40000) + 1000;
-  const bindAddr = Multiaddr(`/ip4/127.0.0.1/udp/${randomPort}`);
+  const bindAddr = `/ip4/127.0.0.1/udp/${randomPort}`;
```

Comment:
Not sure if works here, but usually if you put port "0" it will take random available port